src/Modifiers/CoreModifiers.php

```diff
@@ -1268,6 +1269,10 @@ public function isExternalUrl($value)
             $value = $value->value();
```

Comment:
I think we can just change this line to `$value = $value->url();` instead.

Reply:
Thanks but I mean without the entry condition.

Reply:
Ah, gotcha. Tested locally and that's working great.

Reply:
Awesome, thanks!